src/core/hiwire.h

```diff
@@ -191,9 +192,15 @@ JsArray_New();
 /**
  * Push a value to the end of a JavaScript array.
  */
-errcode
+errcode WARN_UNUSED
 JsArray_Push(JsRef idobj, JsRef idval);
 
+/**
+ * Same as JsArray_Push but panics on failure
```

Comment:
So if I understand correctly, with `JsArray_Push` we catch the JS exception, return the error code check it and then panic on failure.

While this panics directly?

Reply:
Exactly.